src/handlers/response/ResponseHandlers.ts

```diff
@@ -14,7 +16,23 @@ const noContent: ResponseHandler = {
 
 const success: ResponseHandler = {
     canProcess: isAnyOkStatus,
-    process: async (response, responseBodyFormat = 'json') => response[responseBodyFormat](),
+    process: async (response, responseBodyFormat = 'json') => {
```

Comment:
Is it necessary for all users of our library to have this change?
Couldn't you create a ResponseHandler directly in your project?

Reply:
For context: JavaScript will automatically round integers that are outside of its "safe" bounds, i.e., from -(2^53 - 1) to 2^53 - 1, inclusive (±9,007,199,254,740,991). This means we'll lose precision if we convert values beyond those boundaries to integers.

The suggested fix is to return out of bound integers as strings, such that no precision will be lost.

As I mentioned earlier, this should only happen exceptionally, such as when data base ids are returned as integers in an API response body (e.g., `rowid`).

Reply:
Thanks for the explanation, but I think you should have put it directly in the description of your pull request to avoid any misunderstanding.

Reply:
I just backfilled the PR description